networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -112,6 +147,80 @@ TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraphEdgeDeletion) {
 
 }
 
+TEST_P(DynBetweennessGTest, testDynApproxBetweenessGeneratedGraph) {
+    DorogovtsevMendesGenerator generator(200);
+    Graph G1 = generator.generate();
+    Graph G(G1, isWeighted(), isDirected());
+    count n = G.numberOfNodes();
+    if (isWeighted()) {
+        Aux::Random::setSeed(42, false);
+        G.forEdges([&](node u, node v) { G.setWeight(u, v, Aux::Random::probability()); });
+    }
```

Comment:
Since this is used in the other test as well, we could generate random weights in a separate function within `DynBetweennessGTest` instead of duplicating the code.

Reply:
I think that's a good idea, but let's do that in a separate PR.

Reply:
Cool then I am just leaving this piece of code in here for now and then changing it for the next PR.